WakeLockSentinel/WakeLockSentinel_release_method.html

```diff
@@ -0,0 +1,60 @@
+<div>{{draft}}{{securecontext_header}}{{DefaultAPISidebar("Screen Wake Lock API")}}</div>
+
+<p class="summary">The <strong><code>release()</code></strong> method of the {{domxref("WakeLockSentinel")}} interface releases the {{domxref("WakeLockSentinel")}}, returning a {{jsxref("Promise")}} that is resolved once the sentinel has been successfully released.</p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<!-- slightly unsure about this because in practice you most likely won't pass it into a var, just call it -->
```

Comment:
```suggestion
```

Reply:
You were close. Good guess or research.

Reply:
From code examples and my own implementation. It just isn't used like this. I would rather have wakeLock.release().then(...) but wasn't sure as I haven't come across an example without _var_ (yet)